### DISCUSSION TITLE
friends/

### DISCUSSION THREAD

**Discussion by giscus** [songtianlun/songtianlun.github.io#8] [Announcements]
# friends/

友链区 细语呢喃 一个神秘的人 wshunli’s Blog 一个神秘的人 辣椒の酱 一个神秘的人 Rat's Blog 一个神秘的人 LOGI 一个神秘的人 宇宙湾 大数据和人工智能的原创文章

https://www.frytea.com/friends/

**Comment by wangyunzi** [comment]
哇哦博客变化好大啊

**Comment by kiraskyler** [comment]
加油！(到此一游)

**Comment by mi7coder** [comment]
来换友链啦~

名称：程序员米七
地址：https://mi7.net
描述：分享点有价值的 - 程序员米七
头像：https://mi7.net/img/myavatar.jpg

**Comment by yviii-com** [comment]
云帆沧海域名已出， 现在已经博客更换名字为 乙未博客  地址  yvii.cn  请更换

**Reply by songtianlun** [reply]
已更新

**Comment by guoxk-me** [comment]
网站名称:
Guo xk 的博客
LOGO 地址:
https://guoxk.me/avatar.png
网站地址:
https://guoxk.me
网站描述
自言自语的地方

申请友链
